wiki/markdown/usage.md

```diff
@@ -0,0 +1,7 @@
+## Usage
+
+To get information about configuration and usage of **REMReM Generate CLI** see [here](https://eiffel-community.github.io/eiffel-remrem-generate/cliUsage.html).
```

Comment:
The usage pages should also be converted to markdown and be available on master branch. Then these links can point directly to those pages and we can utilize GitHub's version handling of those files with relative links, should the documentation change.